src/extensionApi.ts

```diff
@@ -79,6 +81,41 @@ export class CommandHandler {
         }
     }
 
+    public async debugServer(mode: string, context?: Protocol.ServerState): Promise<Protocol.StartServerResponse> {
+
+        const folderProject = await vscode.window.showOpenDialog({
```

Comment:
Instead of folder you should rather look for workspace folders and if none are configured you can ask the user to open a folder and add it to the workspace

Reply:
How are you able to force the user to open a workspace folder? I looked at the documentation but i found nothing. The only way i can think of is adding a filter to the dialog, but it will create trouble if there is no workspace.